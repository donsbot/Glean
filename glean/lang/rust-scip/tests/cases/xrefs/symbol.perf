{ "@generated": null, "scip.Symbol.1": 6 }