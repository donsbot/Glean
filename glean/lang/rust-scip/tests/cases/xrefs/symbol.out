[
  "@generated",
  { "key": "rust-analyzer cargo basics 0.1.0 (acc)" },
  { "key": "rust-analyzer cargo basics 0.1.0 (n)" },
  { "key": "rust-analyzer cargo basics 0.1.0 f1_go()." },
  { "key": "rust-analyzer cargo basics 0.1.0 sum_f1()." },
  { "key": "rust-analyzer cargo basics 0.1.0 sum_f2()." },
  { "key": "rust-analyzer cargo basics 0.1.0 sum_f3()." }
]